fix:  feedbacks (#3)
succinctlabs/semaphore-gnark-11: README.md

# Guide to the Succinct Trusted Setup Contribution Ceremony

This tool allows users to run an MPC ceremony for generating the proving and verifying keys for the Groth16 protocol used by Succinct [SP1](https://docs.succinct.xyz/docs/sp1/introduction).

> This repo is based on Worldcoin [Semaphore Merkle Tree Batcher](http://github.com/worldcoin/semaphore-mtb/).

Reviewer: ```suggestion
This repo is adapted from Worldcoin's [Semaphore Merkle Tree Batcher](http://github.com/worldcoin/semaphore-mtb/).
```
